.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
```

Comment:
@marchof not sure what you mean - one comment is about `java.locale.providers` that is not part of `MAVEN_OPTS`. Maybe possible to construct whole command in variable step-by-step, but I'm afraid to run into problem with escaping of quotes. Following suggestion of @bjkail below I'm going to split this command on multiple.